packages/aws-cdk-lib/aws-codebuild/lib/environment-type.ts

```diff
@@ -0,0 +1,17 @@
+/**
+ * The type of environment variables to use in a build project.
```

Comment:
Can you clean up this verbiage?  "The build environment type ... "

Reply:
Fixed 👍